Update obfuscation-ui.mds (#10987)
newrelic/docs-website: src/content/docs/logs/ui-data/obfuscation-ui.mdx

Obfuscation has per-minute CPU limits. If your account hits your resource limits, logs won't be obfuscated as expected. To check your CPU limits, go to your [system **Limits** page](/docs/data-apis/manage-data/view-system-limits/#limits-ui) in the New Relic **Data management** UI.

<Callout variant="caution">
  If you exceed the obfuscation per-minute CPU limits and logs cannot be obfuscated or hashed, the logs will be **dropped**. This is to prevent PII or other sensitive data from being ingested inadvertently.
</Callout>

> You can also use the **Health** tab in the **Logs Obfuscation** UI to evaluate how well your obfuscation rules are working, which are being skipped, and whether they need any fine-tuning. Obfuscation rules are CPU intensive, so these charts can help you decide which rules are most impacted by any resource limitations.

Reviewer: ```suggestion
To evaluate how well your obfuscation rules are working and see which ones are being skipped, go to **Logs Obfuscation > Health**. Obfuscation rules are CPU intensive, so these charts can help you decide which rules are most impacted by any resource limitations.
```

Reviewer: @marcelschlapfer What do you think about this revision?
